.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -17,25 +17,25 @@ A clear and concise description of what the bug is.
 
 ### Screenshots
 
-If applicable, add screenshots to help explain your problem.
+<!-- If applicable, add screenshots to help explain your problem. -->
 
 ### Operating System
 
-*Please consider your privacy, before completing this section.*
+<!-- *Please consider your privacy, before completing this section.* -->
 
-Example: Ubuntu 16.04
+<!-- Example: Ubuntu 22.04.3 LTS -->
 
 ### Logs
 
-*Please consider your privacy, before completing this section.*
+<!-- *Please consider your privacy, before completing this section.* -->
 
 ```
 Insert Your Logs Here
 ```
```

Comment:
This could be commented out too.

Reply:
no because the the logs should be pasted there.
when you comment it out, then it's not shown...

Reply:
Most of issue openers don't use any of those, that's the motive for this PR in the first place right?

Reply:
no.
if logs don't get entered (while applicable) that's a problem of the person filling it in "incorrectly", it's not a problem of the template itself